### PR TITLE
initial fonts on ios

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers first pass at the fonts, i'll branch off of this to do it on android next

### CODE REVIEW

File: shared/common-adapters/text.native.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    const terminalPrefix = this._terminalPrefix(this.props.type)
+    const className = (this.props.className || '') + ' ' + (linkClassname || '')
+
+    if (this.props.contentEditable) {
+      return (
+        <RNText
+          ref='text'
+          className={className}
+          style={style}
+          contentEditable
```
Comment (MarcoPolo) [inline shared/common-adapters/text.native.js]: Does contentEditable work in react-native?

**Comment by MarcoPolo** [comment]
:+1:

File: shared/common-adapters/text.native.js (hunk 1 above)
Comment (chrisnojima) [inline shared/common-adapters/text.native.js]: no. but this is only used by input right

Comment (MarcoPolo) [inline shared/common-adapters/text.native.js]: yeah

Comment (chrisnojima) [inline shared/common-adapters/text.native.js]: k, i fully expect us to have to re-touch these files as we expand out the rest of the components

**Pull request merged**